web/public/js/directives/cd-email-users/style.less

```diff
@@ -0,0 +1,46 @@
+.cd-email-users {
+  margin: 0 8px;
+  @border-radius: 10px;
```

Comment:
Variables 👍 
Arbitrary random numbers that don't conform to any kind of grid 👎 
😛 

Personally, I'm partial to base 2 numbers, although I know Bootstrap's margins are `15px`...